libs/SFMLComponents/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+set(SFML_INPUT_INC_ROOT ${PROJECT_SOURCE_DIR}/libs/SFMLComponents)
+
+set(TARGET ${PROJECT_NAME}_SFMLComponents)
+
+set(SRC
+    ${SFML_INPUT_INC_ROOT}/AllowControllerComponent.hpp
+    ${SFML_INPUT_INC_ROOT}/AllowMouseAndKeyboardComponent.hpp
+    ${SFML_INPUT_INC_ROOT}/ControllerButtonInputComponent.hpp
+    ${SFML_INPUT_INC_ROOT}/ControllerJoystickInputComponent.hpp
+    ${SFML_INPUT_INC_ROOT}/KeyboardInputComponent.hpp
+    ${SFML_INPUT_INC_ROOT}/MouseInputComponent.hpp
+    ${SFML_INPUT_INC_ROOT}/ECSActions.hpp
+)
+
+add_library(${TARGET} ${SRC})
```

Comment:
Have you thought about an interface library ? (Only .hpp, so archive is overkilled)

Reply:
mmmh finally I can't, because I use the sfml library as public library. Have you a solution or an archive will be OK @LucasTesnier ?

Reply:
Just unlick the sfml library from the Cmake

It will be automagic after compilation

Reply:
I don't like automagic, but let's try